crates/core/src/lib.rs

```diff
@@ -169,6 +169,9 @@ pub fn default_test_builder() -> Builder {
 
 pub mod factories;
 
+// #[cfg(any(doc, docsrs))]
+pub mod doc;
```

Comment:
why is the attr commented out?

Reply:
Oops, forgot to put that back in. It was commented so that links would be resolved in the IDE.